SDDSThemeBuilder/TOKEN_META.md

```
@@ -0,0 +1,116 @@
+# Мета-информация по токенам (`config-info-tokens-ios.json`)
+
+Генератор `SDDSThemeBuilder` при генерации темы отдаёт мета-файл по всем сгенерированным
+токенам дизайн-системы. Файл нужен внешним инструментам (binding/дизайн-тулинг), чтобы
+однозначно сопоставить исходный токен, сгенерированный Swift-символ и его резолвнутое значение.
+
+## Где лежит результат
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)